Translations/make_translation.py

```diff
@@ -1263,18 +1263,56 @@ def get_translation_sanity_checks_text(defs: dict) -> str:
     return sanity_checks_text
 
 
+def get_version_suffix(ver) -> str:
+    suffix = str("")
+    try:
+        # Use commands _hoping_ they won't be too new for one environments nor deprecated for another ones:
+        ## - get commit id; --short=7 - the shorted hash with 7 digits (increase/decrease if needed!)
+        sha_id = f"{subprocess.check_output(['git', 'rev-parse', '--short=8', 'HEAD']).strip().decode('ascii').upper()}"
+        ## - if the exact commit relates to tag, then this command should return one-line tag name:
+        tag = f"{subprocess.check_output(['git', 'tag', '--points-at', '%s' % sha_id]).strip().decode('ascii')}"
+        ## - get short "traditional" branch name (as in `git branch` for that one with asterisk):
+        branch = f"{subprocess.check_output(['git', 'symbolic-ref', '--short', 'HEAD']).strip().decode('ascii')}"
+        if tag and "" != tag:
```

Comment:
Probably such kind of extra checks is too much but since we work with strings I thought it's better to double-check (not sure it makes sense in Python though).

Reply:
If it has no runtime cost, check as much as possible is my personal belief :)